File: HELLO_JAVA/src/Human.java

public class Human extends Animal {
	String job = "student";
	
	public void chibbo(String job) {
		this.job = job;
	}
	
	public static void main(String[] args) {
		Human hum = new Human();
		System.out.println(hum.x);
		System.out.println(hum.y);
		System.out.println(hum.job);
		hum.move(100, 50);
		hum.chibbo("programmer");
		System.out.println(hum.x);
		System.out.println(hum.y);
		System.out.println(hum.job);
		
	}
}

File: HELLO_VANILLA/human.js
class Human extends Animal{
	constructor() {
		this.job="student";
	  }
	
	chibbo(job){
			this.job = job;
	  }
}


	const hum = new Human();
	console.log("x: " + hum.x);
	console.log("y: " + hum.y);
	console.log(hum.job);
	
	ani.move(100, 50);
	hum.chibbo("programer");
	
	console.log(hum.x);
	console.log(hum.y);
	console.log(hum.job);